gulpfile.js/tasks/design-system.js

```diff
@@ -13,6 +13,7 @@ gulp.task('design-system', ['build:v4'], (done) => {
   runSequence(
     'design-system:generate',
     'copy:design-system',
+    'copy:repository-yaml',
```

Comment:
Can we call this something less specific to the filename? 

`copy:leak-detection-service-config` or similar?